docs/topics/proc_deploying-a-springboot-application-using-war-files.adoc

```diff
@@ -2,10 +2,13 @@
 = Deploying a {SpringBoot} application using WAR files
 // Consider placing this in the Spring Boot runtime documentation instead?
 // You can Package and deploy your application as a war file
+
+ifdef::built-for-spring-boot-1-5[]
 [IMPORTANT]
 --
-Red Hat does not support packaging and deploying {SpringBoot} applications using WAR files in this release of {ProductShortName}.
+Red Hat does not support packaging and deploying {SpringBoot} applications using WAR files in {ProductShortName} {SpringBoot} 1.5.x.
```

Comment:
Rendered, this is _Red Hat does not support packaging and deploying Spring Boot applications using WAR files in Application Development on OpenShift Spring Boot 1.5.x._ That sounds kinda weird.